source/overview/faq.rst

```diff
@@ -69,12 +69,39 @@ To add yourself to the `Mattermost page on LinkedIn <https://www.linkedin.com/co
   - Update my headline: Leave unchecked
   - Description: Leave blank or write a sentence about what you have contributed. 
 
-Mobile Applications
+Notifications
 -------------------
 
+What determines if a desktop notification should be triggered?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Desktop notifications are triggered under to following conditions. Click to expand the flow chart.
+
+.. image:: ../images/desktop_notification_chart.png
+  :width: 239px
+  :height: 232px
+
+What determines if an email notification should be triggered?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Email notifications are triggered under to following conditions. Click to expand the flow chart.
+
+.. image:: ../images/email_notification_chart.png
```

Comment:
Is "or less" accurate for the language about batching? Does the system check in regular intervals on the clock regardless of activity, or does the timer start when a notification is triggered, and then after 15 minutes (or an hour) with no further triggers the batch email is sent? I thought it was the latter, but I'm not 100% sure now.

<img width="610" alt="Screen Shot 2019-10-10 at 11 46 01 AM" src="https://user-images.githubusercontent.com/25826533/66597778-80ab4900-eb54-11e9-83a2-9a6cdff499c9.png">

Reply:
The latter is correct, but if you have a message at T+0, then another at T+12, you'll get a notification at T+15. The "or less" is meant to indicate that if there are more messages after the first, the notification may come sooner

Reply:
Ah okay, I see, thanks.